README.md

```diff
@@ -183,8 +184,24 @@ In order to install the OpenEuropa Multilingual demo module follow [the instruct
 ./vendor/bin/drush en oe_multilingual_demo -y
 ```
 
-## Contributing
+## Enabling URL suffix
+
+This is optional and should be done only if your website should follow the [IPG rules](http://ec.europa.eu/ipg/print/print_all_content/index_en.htm#3.0) for language negotiation.
+
+In order to use this feature you need to enable the OpenEuropa Multilingual URL Suffix module by following [this instructions][10] or via [Drush][11] by running:
+```bash
+./vendor/bin/drush en oe_multilingual_url_suffix -y
+```
+
+After module enabling:
 
+1. Go to `admin/config/regional/language/detection` as admin;
+2. Inside "Interface text language detection" and "Content language detection" disable `URL` method.
+3. Enable `URL suffix` for "Interface text language detection" and "Content language detection";
+4. Make sure that `URL suffix` located on top of list or after `Administration pages` negotiation method
+5. Click on `Configure` for `URL suffix` and configure the language suffixes.
```

Comment:
I haven't checked the actual UI but I'd consider rewriting the piece above as follow:

```
After enabling the module:

1. Go to `admin/config/regional/language/detection`
2. Disable the "URL" method in "Interface text language detection" and "Content language detection"
3. Enable the "URL suffix' method in "Interface text language detection" and "Content language detection";
4. Make sure that the "URL suffix" method is located on top of list but after the "Administration pages" negotiation method, if you use it
5. Save new order
6. If necessary configure the `URL suffix` method by clicking on "Configure"
7. Rebuild cache
```

Reply:
Done.